src/MassTransit/SagaStateMachine/SagaStateMachine/Activities/FaultedScheduleActivity.cs

```diff
@@ -65,7 +65,7 @@ public async Task Faulted<T>(BehaviorExceptionContext<TSaga, T> context, IBehavi
         async Task Schedule<T>(BehaviorExceptionContext<TSaga, T> context, BehaviorExceptionContext<TSaga, TException> exceptionContext)
             where T : Exception
         {
-            Guid? previousTokenId = _schedule.GetTokenId(context.Saga);
+            Guid? previousTokenId = _schedule?.GetTokenId(context.Saga);
```

Comment:
can it be null?

Reply:
It should never be null, would rather check in the constructor than here.